docs/README.md

```diff
@@ -3,53 +3,36 @@
 [![Known Vulnerabilities](https://snyk.io/test/github/commercetools/commercetools-sync-java/4b2e26113d591bda158217c5dc1cf80a88665646/badge.svg)](https://snyk.io/test/github/commercetools/commercetools-sync-java/4b2e26113d591bda158217c5dc1cf80a88665646)
 
- 
-Java library that imports commercetools platform data in the following ways:
-             
- 1. Synchronise data coming from an external system in any form (CSV, XML, etc..) that has been already mapped to 
- [commercetools-jvm-sdk](https://github.com/commercetools/commercetools-jvm-sdk) resource draft objects 
- (e.g. [CategoryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraft.java)).
- 
- 2. Synchronise data from another commercetools project as 
- [commercetools-jvm-sdk](https://github.com/commercetools/commercetools-jvm-sdk) resource draft objects 
- (e.g. [CategoryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraft.java)).
- 
- 
- > Synchronise: Resources will either be created or updated. But they will **not** be deleted.
- 
- ⚡ See the [Quick Start Guide](https://commercetools.github.io/commercetools-sync-java/doc/usage/QUICK_START/) for more information on building a product importer!
- 
- 🔛 Check out the [commercetools-project-sync](https://github.com/commercetools/commercetools-project-sync) for a ready-to-use CLI application that syncs your entire data catalogue between 2 commercetools projects! 
 
-The library supports synchronising the following entities in commercetools
-    
- - [Categories](usage/CATEGORY_SYNC.md)
- - [Products](usage/PRODUCT_SYNC.md)
- - [InventoryEntries](usage/INVENTORY_SYNC.md)
- - [ProductTypes](usage/PRODUCT_TYPE_SYNC.md)
- - [Types](usage/TYPE_SYNC.md)
- - [CartDiscounts](usage/CART_DISCOUNT_SYNC.md)
- - [States](usage/STATE_SYNC.md)
- - [TaxCategories](/docs/usage/TAX_CATEGORY_SYNC.md)
- - [CustomObjects](/docs/usage/CUSTOM_OBJECT_SYNC.md)
- - [Customers](/docs/usage/CUSTOMER_SYNC.md)
- - [ShoppingLists](/docs/usage/SHOPPING_LIST_SYNC.md)
+Java library which allows to import/synchronise (import changes) the data from any arbitrary source to commercetools project.
 
-![commercetools-java-sync-final 001](https://user-images.githubusercontent.com/9512131/31230702-0f2255a6-a9e5-11e7-9412-04ed52641dde.png)
+Supported resources: [Categories](/docs/usage/CATEGORY_SYNC.md), [Products](/docs/usage/PRODUCT_SYNC.md), [InventoryEntries](/docs/usage/INVENTORY_SYNC.md), [ProductTypes](/docs/usage/PRODUCT_TYPE_SYNC.md), [Types](/docs/usage/TYPE_SYNC.md), [CartDiscounts](/docs/usage/CART_DISCOUNT_SYNC.md), [States](/docs/usage/STATE_SYNC.md), [TaxCategories](/docs/usage/TAX_CATEGORY_SYNC.md), [CustomObjects](/docs/usage/CUSTOM_OBJECT_SYNC.md), [Customers](/docs/usage/CUSTOMER_SYNC.md), [ShoppingLists](/docs/usage/SHOPPING_LIST_SYNC.md)
```

Comment:
Can't we avoid the duplication of the documentation in the repo landing page as also in the docs folder?

Reply:
I think not, this duplication is related to mkdocs plugin,  https://github.com/commercetools/commercetools-sync-java/blob/c0cc6c41768bcde703616a1dbde4e2330d3df3c0/gradle-scripts/mkdocs.gradle#L13 
Main readme is not in docs folder.